tests/unittests/test_utilities_dependency.py

```diff
@@ -558,11 +555,11 @@ def test_use_worker_dependencies_disable(self):
             import common_module  # NoQA
 
     @unittest.skipUnless(
-        sys.version_info.major == 3 and sys.version_info.minor in (6, 7, 8),
-        'Test only available for Python 3.6, 3.7, or 3.8'
+        sys.version_info.major == 3 and sys.version_info.minor in (6, 7, 8, 9),
+        'Test only available for Python 3.6, 3.7, 3.8 or 3.9'
     )
-    def test_use_worker_dependencies_default_python_36_37_38(self):
-        # Feature should be disabled in Python 3.6, 3.7, and 3.8
+    def test_use_worker_dependencies_default_python_36_37_38_39(self):
```

Comment:
Why do we need to explicitly add this check? Why not do a simpler check

`sys.version_info.major == 3 and sys.version_info.minor != 10`

Reply:
Or better, use the below option of skipIf

```python
@unittest.skipIf(sys.version_info.major == 3 and sys.version_info.minor == 10,
                 'Test not available for python 3.10')
```

Reply:
This check was to make sure the dependency isolation is not enabled for other versions. Thats why I didnt use a skipif. Changed the condition to !=10.